README.md

```diff
@@ -1,36 +1,37 @@
 # 🛠 tdex-sdk
+
```

Comment:
I guess you have some sort of markdown linter. Please deactivate

Reply:
done